chapter_preliminaries/calculus.md

```diff
@@ -347,8 +347,7 @@ has variables $x_1, x_2, \ldots, x_n$,
 i.e.,  $\mathbf{u} = g(\mathbf{x})$.
 Then the chain rule states that
 
-$$\frac{dy}{dx_i} = \frac{dy}{du_1} \frac{du_1}{dx_i} + \frac{dy}{du_2} \frac{du_2}{dx_i} + \cdots + \frac{dy}{du_m} \frac{du_m}{dx_i} \text{ and thus }
-\nabla_{\mathbf{x}} y = \langle \nabla_{\mathbf{u}} y, \nabla_{\mathbf{x}} \mathbf{u}\rangle.$$
+$$\frac{\partial y}{\partial x_{i}} = \frac{\partial y}{\partial u_{1}} \frac{\partial u_{1}}{\partial x_{i}} + \frac{\partial y}{\partial u_{2}} \frac{\partial u_{2}}{\partial x_{i}} + \ldots + \frac{\partial y}{\partial u_{m}} \frac{\partial u_{m}}{\partial x_{i}} \text{ and thus } (\nabla_{\mathbf{u}} y)^{T} (\nabla_{\mathbf{x}} \mathbf{u}).$$
```

Comment:
\nabla_{\mathbf{x}} y = (\nabla_{\mathbf{u}} y)^\top (\nabla_{\mathbf{x}} \mathbf{u})

Reply:
I agree with @jbritton6, this is indeed an error. The dot product can only be used for a specific x_i, but please fix the equation following the suggestion. Thank you very much for the contribution!

Reply:
Thanks! 
Since the second line in the following should be a row vector (not a column vector), 
![Screen Shot 2022-05-12 at 2 04 42 PM](https://user-images.githubusercontent.com/22279212/168167879-c7b6839b-0c5b-42e7-9d40-9d762577308e.png)

let me know if the following looks better:

![Screen Shot 2022-05-12 at 2 06 56 PM](https://user-images.githubusercontent.com/22279212/168168275-ee2b77b4-5346-4107-b488-586aff36fac7.png)